packages/types/config/runtime.d.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * NuxtConfigurationRuntimeConfig
+ * Documentation : TBD
+ * NuxtRuntimeConfig interface can be extended by users to enable intellisense on $config
+ */
+
+export interface NuxtRuntimeConfig {
+  [key: string]: any
```

Comment:
Care to explain the reasoning for going back to this implementation?

Reply:
It was an internal talk with @kevinmarrec. The issue is that with explicit types, typescript will always error when if using optional chaining (but works fine with `any`) and it is just about type checking.